meta/recipes-core/meta/cve-update-nvd2-native.bb

```diff
@@ -128,6 +129,10 @@ def nvd_request_next(url, api_key, args):
 
     full_request = url + '?' + data
 
+    codes_to_retry = {
+        503, # Service Unavailable
```

Comment:
I think we should also retry on other [5xx](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#:~:text=Server%20error%20responses%20(500%20%E2%80%93%20599)) errors like 500, 502, 504, 507.

Reply:
That makes sense (and is why I arranged for the codes to reside in a set). Is there a canonical list somewhere of status codes that indicate retrying is a good idea? I'd suspect we wouldn't want all the 5xx codes. (I could look at the various RFCs myself if we don't know of one.)

Reply:
Ya I don't think we'd want to try all 5xx errors - only errors where there is a possibility of server recovering. There is some guidance on which ones to retry for [here](https://cloud.google.com/workflows/docs/reference/syntax/retrying).

Reply:
Posted a follow-up PR.